README.md

```diff
@@ -36,25 +36,39 @@ output at the same time it applies them for applications output:
 9. Output Limiter (Ladspa Fast Lookahead Limiter)
 10. Spectrum Analyzer (Gstreamer)
 
-## Installation
+## Build and install
 
 Users upgrading from 1.x to 2.x will have to rebuild their presets. Since
 version 2.0.0 PulseEffects uses a different format. This
 change was necessary to support presets for microphone processing.
 
+```
+$ meson _build --prefix=/usr
+$ cd _build
+$ sudo ninja install
+```
+
```

Comment:
I fundamentally disagree with adding this information to the top and mixing build and regular install instructions. Issues with this section and why it should  be removed:

- We should not expect most users to build and install this.
- This replicates information already present in the wiki.
- This information is incomplete, lacking instructions for installing dependencies etc which, again, is already present in the wiki.

Minor note for the future, in English grammar the title should be "Build and Install".